source/Grid/Grid.test.js

```diff
@@ -120,43 +120,49 @@ describe('Grid', () => {
   describe('shows and hides scrollbars based on rendered content', () => {
     const scrollbarSize = getScrollbarSize()
 
+    function renderTwice (markup) {
```

Comment:
In first render, overflowX or Y always won't be set, so there's noting to test.
The purpose of call forceUpdate() is testing the second render's result.
Or there's other way to do this?

Or we should `adding a fallback to render to handle a potentially undefined size (eg const scrollbarSize = this._scrollbarSize || 0)` as you say?
But this will cause the first render got wrong result about overflowX / overflowY.

sorry, my english is not very well. :)